GoedeDoelenHelpen/ClientApp/src/app/home/home.component.ts

```diff
@@ -1,10 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { NavMenuService } from '../nav-menu/nav-menu.service';
 
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
-  styleUrls: ['./home.component.css']
+  styleUrls: ['./home.component.scss']
 })
 
-export class HomeComponent {
+export class HomeComponent implements OnInit {
+
+  constructor(private service: NavMenuService) {
```

Comment:
mostly is the convention to name the service property just by its name. In the futere you possible need more than one service in a componenten so ```private navMenuService: NavMenuService```